CHANGELOG.md

```diff
@@ -26,8 +26,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/report/compare/release-3.11.4...master
 
-### Fixed
-- Restore missing log label in failed regression test
+### Changed
+- Change API: replace RelativeNonparametricPerformanceJudge constructor with builder
+- Improve failed action report message in `RelativeNonparametricPerformanceJudge`
```

Comment:
JPERF-832 already had 3 pull requests: https://github.com/atlassian/report/pull/64, https://github.com/atlassian/report/pull/67, https://github.com/atlassian/report/pull/68. I wouldn't spend much energy on it, because `RelativeNonparametricPerformanceJudge` will probably get deprecated anyway.

Reply:
Maybe it will, maybe it will not. We don't know the future and how long it's going to stay with us :)
While it's with us I'd like to have it easier to test and understand

Reply:
Fair point, I respect that perspective. It's true that "temporary stuff" lives way longer than we'd like to :smile: